kiva/fonttools/font_manager.py

```diff
@@ -670,6 +671,27 @@ def afmFontProperty(fontpath, font):
 
     return FontEntry(fontpath, name, style, variant, weight, stretch, size)
 
+def extractTTC(fpath):
+    """
+    Extract a list of ttfFontProperty objects from a ttc file
+    """
+    props = []
+    try:
```

Comment:
I'd be tempted to put this try-except in `createFontList` below - have this function either return a correct list or fail; but the code in `createFontList` has to execute a loop and so it shouldn't fail on a particular loop and should be what worries about catching errors.

Reply:
Makes sense to me. Fixed in 533401c